WordPress/Classes/ViewRelated/Reader/ReaderPostCardCellViewModel.swift

```diff
@@ -0,0 +1,149 @@
+
+struct ReaderPostCardCellViewModel {
+
+    // MARK: - Properties
+
+    var siteIconURL: URL? {
+        let scale = WordPressAppDelegate.shared?.window?.screen.scale ?? 1.0
+        let size = ReaderPostCardCell.Constants.iconImageSize * scale
+        return contentProvider.siteIconForDisplay(ofSize: Int(size))
+    }
+
+    var siteTitle: String? {
+        if let post = contentProvider as? ReaderPost, post.isP2Type(), let author = contentProvider.authorForDisplay() {
+            let strings = [author, contentProvider.blogNameForDisplay?()].compactMap { $0 }
+            return strings.joined(separator: " ▸ ")
+        }
+
+        return contentProvider.blogNameForDisplay?()
+    }
+
+    var postDate: String? {
+        guard let dateForDisplay = contentProvider.dateForDisplay()?.toShortString() else {
+            return nil
+        }
+        return siteTitle != nil ? "• \(dateForDisplay)" : dateForDisplay
```

Comment:
Same RTL issue as the above, but in this case, you probably just need to place the dot at the end on RTL layout.

Reply:
Fixed with https://github.com/wordpress-mobile/WordPress-iOS/pull/21687/commits/1d194613f22a5ad67508f3521ac982256cf19cad. Thanks for the catch!

Example:
![rtl](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/e87d9391-6641-4832-8192-a2d58e6f3e72)